changefeedtrigger/readme.md

```diff
@@ -0,0 +1,55 @@
+# README
+The purpose of this sample is to create a log of all new events that occur in a storage account with multidimensional filtering that runs on a timer using an Azure Function
```

Comment:
>The purpose o [](start = 0, length = 13)

rename this readme.md to upper case

Reply:
In my program it is named README.md and I have tried to repush it a few times but for whatever reason once I push it it uploads to git as readme.md. I am not sure why but I will try again today to fix it. Maybe I should change the name and commit it and then name it README again and commit and push and then it will change?

Reply:
https://www.hanselman.com/blog/GitIsCasesensitiveAndYourFilesystemMayNotBeWeirdFolderMergingOnWindows.aspx

Reply:
thank you kamil!